tests/functional/aws-node-sdk/test/object/unsupportedQueries.js

```diff
@@ -0,0 +1,39 @@
+import assert from 'assert';
+import makeRequest from '../../lib/utility/makeRequest';
+
+// Queries supported by AWS that we do not currently support.
+const unsupportedQueries = [
```

Comment:
👍 Yeah, that was the intent behind moving it to routeUtils, I forgot to modify the test file. I switched the location to constants.js though.